packages/pyright-internal/src/localization/localize.ts

```diff
@@ -153,22 +145,8 @@ function getLocaleFromEnv() {
     return defaultLocale;
 }
 
-function loadDefaultStrings(): StringLookupMap {
-    const defaultStrings = loadStringsFromJsonFile(defaultLocale);
-    if (defaultStrings) {
-        return defaultStrings;
-    }
-    console.error('Could not load default strings');
-    return {};
-}
-
-function loadStringsForLocale(locale: string): StringLookupMap {
-    if (locale === defaultLocale) {
-        // No need to load override if we're using the default.
-        return {};
-    }
```

Comment:
Removed because we want to overlay our simplified messages all the time.

Perhaps we should optimise the English path by generating a combined bundle but not sure this is worth it now.